src/components/Dropdown.tsx

```diff
@@ -1,19 +1,24 @@
-import iconArrowDown from '/src/assets/ic_arrow_down.svg';
+import iconArrowDown from '@assets/images/ic_arrow_down.svg';
 import { useState } from 'react';
+import '@assets/styles/Dropdown.css';
+import iconSort from '@assets/images/ic_sort.svg';
+import { useResponsiveApi } from '@hooks/Responsive';
 
-import '/src/styles/Dropdown.css';
-
-import iconSort from '/src/assets/ic_sort.svg';
-
-import { useResponsiveApi } from '/src/Responsive';
+interface DropdownProps {
+  className?: string;
+  first: string;
+  second: string;
+  firstClick?: any;
```

Comment:
아래에서 보니까 firstClick은 onClick에 들어가고있네요! 그러면 함수이고 특히 이벤트 핸들러 함수라는 것을 알 수 있어요!
그러면 ```suggestion
  firstClick?: (event: React.MouseEvent) => void;
```
혹은 
```suggestion
  firstClick?: React.MouseEventHandler<HTMLLIElement>;
```
등으로 할 수 있을 것 같아요.

그리고 핸들러 함수의 경우 props로 받을 때는 `onClick` 처럼 on을 앞에 붙여주는 편입니다.
또한 first가 뭔지 정확히 알기 어려워서 조금 더 명확한 네이밍이 필요할 것 같아요!